synapseclient/annotations.py

```diff
@@ -228,6 +228,127 @@ def to_submission_status_annotations(annotations, is_private=True):
     return synapseAnnos
 
 
+def to_submission_annotations(
+    id: typing.Union[str, int],
+    etag: str,
+    annotations: typing.Dict[str, typing.Any],
+    logger: typing.Optional[typing.Any] = None,
+) -> typing.Dict[str, typing.Any]:
```

Comment:
Discussion: Should SubmissionAnnotations be a class structure or is that not needed?

Reply:
I think we already have an annotation class that can be used here: https://github.com/Sage-Bionetworks/synapsePythonClient/blob/develop/synapseclient/models/annotations.py#L18